modules/ingest-geoip/src/test/java/org/elasticsearch/ingest/geoip/stats/GeoIpDownloaderStatsSerializingTests.java

```diff
@@ -11,29 +11,87 @@
 import org.elasticsearch.common.io.stream.Writeable;
 import org.elasticsearch.test.AbstractXContentSerializingTestCase;
 import org.elasticsearch.xcontent.ConstructingObjectParser;
+import org.elasticsearch.xcontent.ParseField;
 import org.elasticsearch.xcontent.XContentParser;
 
 import java.io.IOException;
+import java.util.List;
 
 public class GeoIpDownloaderStatsSerializingTests extends AbstractXContentSerializingTestCase<GeoIpDownloaderStats> {
 
-    private static final ConstructingObjectParser<GeoIpDownloaderStats, Void> PARSER = new ConstructingObjectParser<>(
+    @SuppressWarnings("unchecked")
+    private static final ConstructingObjectParser<GeoIpDownloaderStats, Void> GEOIP_STATS_PARSER = new ConstructingObjectParser<>(
         "geoip_downloader_stats",
-        a -> new GeoIpDownloaderStats((int) a[0], (int) a[1], (long) a[2], (int) a[3], (int) a[4], a[5] == null ? 0 : (int) a[5])
+        a -> new GeoIpDownloaderStats(
+            (int) a[0],
+            (int) a[1],
+            (long) a[2],
+            (int) a[3],
+            (int) a[4],
+            a[5] == null ? 0 : (int) a[5],
+            a[6] == null ? null : (List<DownloadedDatabaseInfo>) a[6]
```

Comment:
My IntelliJ doesn't like this `null` here -- if this were to run, then I think we'd NPE in the `GeoIpDownloaderStats` constructors.